css/style.css

```diff
@@ -79,6 +106,77 @@ footer {
 /* ADDITIONAL STYLES
 /*******************************************/
 
+.slogan {
+  font-size: 24px;
+  text-align: center;
+  font-weight: 600;
+  color: white;
+  text-align: center;
+  padding-top: 60px;
+}
+
+.button1 {
+  background-color: #f9e42e; /*yellow*/
+  border: none;
+  color: #033048; /*dark blue*/
+  padding: 15px 50px;
+  text-decoration: none;
+  font-size: 16px;
+  font-weight: 600px;
+}
+
+.benefits {
+    background-color: #f0efef;
+    text-align: left;
+    position: relative;
+    padding: 30px;
+}
+.benefits p {
+    font-size: 13px;
+  font-weight: 300px;
+}
+.benefits ul {
+  font-size: 13px;
+  font-weight: 300px;
+
+}
+.testimonial {
+    background-color: #f0efef;
+    text-align: center;
+    position: relative;
+    background: url(../images/testimonial_bg.jpg);
+    color: white;
+        padding: 30px;
+}
+.sign-up{
+  background-color: #f9e42e;
+  text-align: center;
+  padding: 15px;
+}
+.button2 {
```

Comment:
try to come up with more descriptive names to your button classes. You could go with content specific classes such as `.get-it-now-button` or you could could make them more generic like `.primary-button` and `.alternate-button` or `.secondary-button`. etc.